app/views/container_topology/show.html

```diff
@@ -0,0 +1,155 @@
+<!doctype html>
+<html class="no-js">
+<head>
+  <meta charset="utf-8">
+
+  <meta name="viewport" content="width=device-width">
+  <style>
+    /*@font-face {*/
+      /*font-family: 'FontAwesome';*/
+      /*src: url('../assets/fonts/fontawesome-webfont.svg') format('svg');*/
+    /*}*/
+    * {
+      box-sizing: border-box;
+      font-family: sans-serif;
+      font-size: 15px;
+    }
+    body {
+      margin: 0px;
+    }
+    kubernetes-topology-graph {
+      border: 10px solid lightgray;
+      position: fixed;
+      height: 76%;
+      width: 98%;
+      padding: 10px;
+    }
+    .legend {
+      display: inline-block;
+      padding: 0 10px;
+      font-family: "FontAwesome";
+
+    }
+    .legend label {
+      font-weight: 400;
+    }
+
+    .refresh {
+      display: inline-block;
+      width:35px;
+      height:35px;
+      padding:0;
+    }
+
+    #selected {
+      float: right;
+      display: block;
+      margin-top: 15px;
+    }
+
+    kubernetes-topology-icon {
```

Comment:
> @abonas Can you remove the kubernetes prefix from these classes?

they are named like that in the third party component which is used here
https://github.com/kubernetes-ui/topology-graph/blob/master/topology-graph.css